soc/arm/nordic_nrf/nrf53/soc.c

```diff
@@ -120,6 +129,18 @@ static int nordicsemi_nrf53_init(const struct device *arg)
 	nrf_regulators_dcdcen_vddh_set(NRF_REGULATORS, true);
 #endif
 
+#if defined(CONFIG_SOC_NRF5340_CPUAPP) && \
+	DT_HAS_COMPAT_STATUS_OKAY(nordic_nrf_gpio_forwarder)
+	static const int pf_gpios[] = {
```

Comment:
`uint8_t` should be sufficient. And perhaps `pf_psels` would be a more adequate name for the array.

Reply:
Done.